openqa-mail-notification/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM registry.opensuse.org/home/vpereirabr/dockerimages/containers/obs-tools/base:latest
+
+RUN zypper --non-interactive --quiet refresh \
+ && zypper --non-interactive install ruby2.5-rubygem-mail
+
+RUN chown --recursive frontend:users /home/frontend/
+
+WORKDIR /home/frontend/openqa-mail-notification
+
+USER frontend
+
+ENTRYPOINT ["./entrypoint.sh"]
+
+CMD ["./mail_notification.rb"]
```

Comment:
```ruby
ENTRYPOINT ["/usr/local/bin/entrypoint.sh"]

CMD ["/usr/local/bin/mail_notification.rb"]
```

Reply:
Done

Reply:
I've had to undo this change...

Reply:
This change works after applying: https://github.com/openSUSE/obs-tools/pull/32/files#r364749904

Reply:
Yes I figured it out this morning. Thanks :D

Reply:
I pushed a new commit with the absolute paths. Now its fine.